policy/modules/system/sysnetwork.if

```diff
@@ -671,6 +709,42 @@ interface(`sysnet_search_dhcp_state',`
 	allow $1 dhcp_state_t:dir search_dir_perms;
 ')
 
+########################################
+## <summary>
+##  Create an object in the pid directory
+##  with an automatic type transition to the
+##  network config type.
+## </summary>
+## <param name="domain">
+##  <summary>
+##  Domain allowed access.
+##  </summary>
+## </param>
+## <param name="private type">
+##  <summary>
+##  The type of the object to be created
+##  </summary>
+## </param>
+## <param name="object">
+##  <summary>
+##  The object class of the object being created.
+##  </summary>
+## </param>
+## <param name="name" optional="true">
+##  <summary>
+##  The name of the object being created.
+##  </summary>
+## </param>
+#
+interface(`sysnet_pid_filetrans_config', `
+	gen_require(`
+		type net_conf_t;
+	')
+
+	files_pid_filetrans($1, net_conf_t, $2, $3, $4)
+	init_daemon_pid_file(net_conf_t, $2, $3)
```

Comment:
Not acceptable, but I don't think would be necessary anymore in light of the previous comments.